packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.js

```diff
@@ -0,0 +1,83 @@
+import { Box, DropButton } from 'grommet'
+import { Filter } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React, { useRef, useState } from 'react'
+
+import Characteristics from '../Characteristics'
+import FilterButton from '../Characteristics/components/FilterButton'
+
+export default function FilterStatus (props) {
+  const { task } = props
+  const filterStatusRef = useRef()
+
+  // TODO: refactor filter state to model
+  const [ filters, setFilters ] = useState({})
+
+  function handleFilter (characteristicId, valueId) {
+    let newFilters = Object.assign({}, filters)
+    if (valueId) {
+      newFilters[characteristicId] = valueId
+    } else if (characteristicId) {
+      delete newFilters[characteristicId]
+    } else {
+      newFilters = {}
+    }
+    setFilters(newFilters)
+  }
+
+  const selectedCharacteristicIds = Object.keys(filters)
+
+  return (
+    <Box
+      ref={filterStatusRef}
+      align='center'
+      direction='row'
+      fill='horizontal'
+    >
+      <DropButton
+        icon={<Filter />}
```

Comment:
Intended icon is in InVision, but I'm blanking on how best to incorporate, is there a similar situation (custom icon) I could reference? I attempted copying svg info from InVision and using <Blank />, but did something wrong.

Reply:
Check out an example here: https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/MoveButton/MoveIcon.js

And @beckyrother may be able to get you the SVG markup if it's not downloading out of invision. 

This could be updated in the follow up design pass we've talked about.